src/utils/replace.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 Christopher Sugai.
+// Licensed under the MIT license (http://opensource.org/licenses/MIT)
+// This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! 
+//! Traits to replace Ns with pseudorandom nucleotide bases and convert U to/from T.
+//! The methods in this trait assumes that:
+//! - The sequence contains only ASCII characters.
+//! - Lowercase and uppercase characters are not mixed.
```

Comment:
It actually assumes that everything is uppercase, right? The trait implementations below only replace uppercase T/U/N with uppercase U/T/*